apps/backoffice-v2/src/lib/blocks/create-blocks-typed/types.ts

```diff
@@ -214,6 +216,14 @@ export type TDialogCell = {
   value: ComponentProps<typeof Dialog>;
 };
 
+export type TProcessTrackerBlock = {
+  type: 'processTracker';
+} & IProcessTrackerProps;
+
+export type TNodeBlock = {
+  type: 'nodeBlock';
+} & INodeBlockProps;
```

Comment:
These types should be the source of truth, the cells themselves should use `ExtractCellProps`.